src/contexts/dices.jsx

```diff
@@ -38,13 +38,24 @@ const DicesProvider = ({ children }) => {
     return dice
   }
 
+  const remove = id => (e) => {
+    e.stopPropagation()
```

Comment:
why is this needed ?

Reply:
I forget to remove it.
In the first version I wrote I added the remove button in the dice button.

But it was not clever.
I changed to have a `div` representing the whole dice, with "roll" button + "remove" button as siblings, but I forgot to remove this.